pkg/rule/api/v1.go

```diff
@@ -92,7 +92,7 @@ func (api *API) rules(r *http.Request) (interface{}, []error, *qapi.ApiError) {
 				enrichedRule = alertingRule{
 					Name:                    rule.Name(),
 					Query:                   rule.Query().String(),
-					Duration:                rule.Duration().Seconds(),
+					Duration:                rule.HoldDuration().Seconds(),
```

Comment:
Hold?

Reply:
The method was changed upstream. There is no `Duration()` now. I will verify this again.

Reply:
Thanks!

Reply:
Yes, the name was changed, that's all.